dev.docker-compose.yml

```
@@ -6,6 +6,8 @@ services:
       service: backend
     environment:
       - DB_URL=postgresql://user:pass@db/answerbook
+      - MATHPIX_APP_ID=${MATHPIX_APP_ID}
+      - MATHPIX_APP_KEY=${MATHPIX_APP_KEY}
```

This should be equivalent to 

```suggestion
      - MATHPIX_APP_ID
      - MATHPIX_APP_KEY
```

Where do these env vars come from? Is the way to obtain them documented somewhere?

---

Added info about setting this up to the README in cd8d75174fc747ed01be4192cfcfc15084e24e32

(note that we didn't obtain the API keys ourselves, so we can't provide documentation as to how to obtain them from mathpix)

---

So where did they come from? Asking just out of curiosity at this point

---

Jason gave us the API keys...